batchflow/dsindex.py

```diff
@@ -34,12 +39,48 @@ def from_index(cls, *args, **kwargs):
 
     @classmethod
     def concat(cls, *index_list):
-        """ Create index by concatenating other indices. """
+        """Create index by concatenating other indices.
+
+        Parameters
+        ----------
+        index_list : list
+            indices to be concatenated. each item in the list should
+            have method .index, that returns an 1-d array-like structure
+
+        Returns
+        -------
+        DatasetIndex with one common index
+        """
         return DatasetIndex(np.concatenate([i.index for i in index_list]))
 
     @staticmethod
     def build_index(index):
-        """ Check index type and structure. """
+        """ Check index type and structure.
+
+        Parameters
+        ----------
+        index : int, 1-d array-like or callable
+            defines structure of DatasetIndex
+
+            - list, numpy.array, pandas.DataSeries - structure is np.array
+
+            - int - structure is np.arange() of given length
+
+            - callable - a function which returns data that can
+                be represented as array
+
```

Comment:
One can hardly understand what this text means. Try to reformulate it more clearly